MinecraftClient/Protocol/Handlers/Protocol16.cs

```diff
@@ -845,5 +845,10 @@ public static bool doPing(string host, int port, ref int protocolversion)
             }
             catch { return false; }
         }
+
+        public bool SelectTrade(int selectedSlot)
+        {
+            return false; //Currently not implemented
```

Comment:
Maybe don't add `SelectTrade` to the ChatBot API for now if it's not implemented?

Reply:
It's only 1.13+ and implemented in protocol18, how would I make it so that it doesn't throw errors if I don't have it in Protocol16?

Reply:
My bad, sorry